packages/document/docs/en/guide/basic/custom-page.mdx

```diff
@@ -97,6 +96,49 @@ Here are some commonly used global styles:
 
 > If you want to know more about the internal global styles, you can check [vars.css](https://github.com/web-infra-dev/rspress/blob/main/packages/core/src/theme-default/styles/vars.css)
 
+### Tailwind CSS
+
+In order to get Tailwind CSS working with Rspress, you can use the following steps:
+
+1. Install Tailwind CSS and its dependencies:
+
+<PackageManagerTabs command="install tailwindcss postcss autoprefixer -D" />
```

Comment:
we do not need to install `postcss` and `autoprefixer`, as Rsbuild supports them by default 😄

```suggestion
<PackageManagerTabs command="install tailwindcss -D" />
```

Reply:
see: https://rsbuild.dev/guide/basic/tailwindcss